docs/develop/showcase.md

```diff
@@ -0,0 +1,96 @@
+---
+id: showcase
+title: Showcase
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+
+
+## Wallets
+
+Wallets are like the swiss army knife of the web3.
+Your wallet will store private information like your secret for your DID or your KILT credentials.
+Each time a website wants to know something about you, the wallet will ask for your permission to share or sign information.
+
+
+<a href="https://sporran.org">
+    <ThemedImage
```

Comment:
uhhh, nice dark mode support

Reply:
The junior CSS engineer is slowly understanding the advanced concepts 😜